modules/ztp-configuring-ptp-fast-events.adoc

```diff
@@ -4,38 +4,42 @@
 
 :_module-type: PROCEDURE
 [id="ztp-configuring-ptp-fast-events_{context}"]
-= Configuring PTP fast events using PolicyGenTemplate CRs
+= Configuring PTP fast events with PolicyGenTemplate CRs
 
-You can configure PTP fast events for vRAN clusters that are deployed using the GitOps Zero Touch Provisioning (ZTP) pipeline. Use `PolicyGenTemplate` custom resources (CRs) as the basis to create a hierarchy of configuration files tailored to your specific site requirements.
+You can configure PTP fast events on managed clusters that you deploy with the GitOps Zero Touch Provisioning (ZTP) pipeline.
 
 .Prerequisites
 
 * Create a Git repository where you manage your custom site configuration data.
```

Comment:
add local-storage-operator as a prerequisite.

Reply:
pre-req on the managed cluster or on the hub cluster?